src/serialize/column.rs

```diff
@@ -186,6 +188,10 @@ where
     S: Serializer,
     C: SerializeContext,
 {
+    if !world.is_flushed() {
+        return Result::Err(<S as Serializer>::Error::custom("Serialize requires the world to be flushed first."));
```

Comment:
It's not obvious why this is necessary, or whether it's even necessary at all. Recall that most users don't need serialization to preserve entity allocator state to begin with.

Reply:
Removed.